launchable/utils/authentication.py

```diff
@@ -36,6 +37,24 @@ def authentication_headers():
     if token:
         return {'Authorization': 'Bearer {}'.format(token)}
 
+    if os.getenv('EXPERIMENTAL_GITHUB_OIDC_TOKEN_AUTH'):
+        req_url = os.environ['ACTIONS_ID_TOKEN_REQUEST_URL']
+        rt_token = os.environ['ACTIONS_ID_TOKEN_REQUEST_TOKEN']
+        if not req_url or not rt_token:
```

Comment:
[must]
`os.environ[]` may raise key error if the keys are not present. 
Better to use `os.getenv()` which will return None if the key is absent.

Reply:
Done